seaborn/0.12.2/Dockerfile

```diff
@@ -0,0 +1,31 @@
+################## BASE IMAGE ######################
+
+FROM python:3.10.8
+
+################## METADATA ######################
+
+LABEL base_image="python:3.10.8"
+LABEL version="2"
+LABEL about.summary="Python library for plotting."
+LABEL software="seaborn"
+LABEL software.version="0.12.2"
+LABEL about.tags="implemented-in::python"
+LABEL about.home="https://seaborn.pydata.org"
+LABEL about.software="https://seaborn.pydata.org"
+LABEL about.documentation="https://seaborn.pydata.org"
+LABEL about.license="BSD-3-Clause"
+LABEL about.license_file="https://github.com/mwaskom/seaborn/LICENSE.md"
+LABEL extra.identifiers.biotools="seaborn, pandas, numpy, matplotlib"
```

Comment:
should be a single tool link to a biotools identifier. In this case I do not find any biotools entry for seaborn, so should be left empty

Reply:
I am leaving pandas and python here, is it fine?

Reply:
or there should not be anything?

Reply:
Sorry I had misunderstood what should be here.
Both comments should be fixed now.
Thank you for the super fast review!